docs/sharepoint/extensiondata-element.md

```diff
@@ -49,7 +49,7 @@ ms.workload:
 
 |||
 |-|-|
-|**Namespace**|http<nolink>://schemas.microsoft.com/VisualStudio/<br>2010/SharePointTools/SharePointProjectItemModel|
```

Comment:
If we get rid of nolink, does the link become clickable?

Reply:
No, because I escaped the `http://` to `http:\/\/`